index.html

```diff
@@ -413,55 +413,78 @@ <h3>Proofs</h3>
         </p>
 
         <p>
-A <a>data integrity proof</a> typically includes at least the following
-attributes:
+The following attributes are defined for use in a <a>data integrity proof</a>:
         </p>
 
         <dl style="margin-left: 1em;">
-          <dt><dfn>type</dfn><dt>
+          <dt><dfn>type</dfn></dt>
           <dd>
-<em>Required</em>. The specific <a>proof type</a> used. For example, an
-<code>Ed25519Signature2020</code> type indicates that the proof includes a
-digital signature produced by an <code>ed25519</code> cryptographic key.
+The specific <a>proof type</a> used for the cryptographic proof MUST be
+specified as a string that maps to a URL [[URL]]. Examples of proof types
+include <code>DataIntegrityProof</code> and <code>Ed25519Signature2020</code>.
+Proof types determine what other fields are required to secure and
+verify the proof.
           </dd>
 
-          <dt><dfn class="lint-ignore">proofPurpose</dfn><dt>
+          <dt><dfn class="lint-ignore">proofPurpose</dfn></dt>
           <dd>
-<em>Required</em>. The specific intent for the proof, the reason why an entity
-created it. Acts as a safeguard to prevent the proof from being misused for a
-purpose other than the one it was intended for. For example, a proof can be used
-for purposes of <code>authentication</code>, for asserting control of a
-Verifiable Credential (<code>assertionMethod</code>), and several others.
+The reason the proof was created MUST be specified as a string that maps to a
+URL [[URL]]. The proof purpose acts as a safeguard to prevent the proof from
+being misused for a purpose other than the one that was intended. For example, a
+proof can be created for the purpose of <code>authentication</code> or for
+asserting a set of statements such as those in a Verifiable Credential
+(<code>assertionMethod</code>).
```

Comment:
It would be good to also include an example of misuse/misapplication and/or the harm potentially caused thereby.

Reply:
Looks good at a quick read. We might try to take a page from existing X509 certs, which can be set to be usable for things like "use to sign email", "use to encrypt email", "use for login (e.g., with ssh)", etc. -- and the mis-step is clear in the error messages when you try to use an "email signer" cert for an ssh login.